src/main/java/com/exasol/adapter/dynamodb/mapping/TableMappingDefinition.java

```diff
@@ -16,20 +16,20 @@
     private static final long serialVersionUID = 3568807256753213582L;
```

Comment:
This also should be updated. Please also update it in all updated classes that has serial uid.